docs/public/operators-on-minikube.adoc

```diff
@@ -0,0 +1,204 @@
+----
+
+Give it a few seconds before checking the availability of the etcd Operator:
+[source,sh]
+----
+$ odo catalog list services
+Operators available in the cluster
+NAME                                CRDs
+etcdoperator.v0.9.4-clusterwide     EtcdCluster, EtcdBackup, EtcdRestore
+----
+
+==== Troubleshooting
+
+If you don't see etcd Operator using above command or by doing `kubectl get csv -n operators`, make sure that pod belonging to the `CatalogSource` named `operatorhubio-catalog` is running:
+[source,sh]
+----
+$ kubectl get po -n olm | grep operatorhubio-catalog
+----
+
+If the state of this pod is `CrashLoopBackOff`, delete it so that Kubernetes will automatically spin up a new pod for the `CatalogSource`:
+
+[source,sh]
+----
+$ kubectl delete po -n olm <name-of-operatorhubio-catalog-pod>
+----
+
+Once the pod for this `CatalogSource` is up, wait a few seconds before trying to find the etcd Operator when you do `odo catalog list services`.
+
+=== Installing the Service Binding Operator
+
+We use the link:https://github.com/redhat-developer/service-binding-operator/[Service Binding Operator] to provide `odo link` feature which links an odo component with an Operator backed service. Thus, to be able to use this feature, it's essential that we install the Operator first.
+
+Service Binding Operator is not yet available via the OLM. The team is link:https://github.com/redhat-developer/service-binding-operator/issues/727[working on making it available] through OLM.
+
+At the moment, to make Service Binding Operator work on Kubernetes, we need to keep a couple of things in mind:
+
+. It needs to be installed by cloning its GitHub repository
+. It must be installed in an individual namespace unlike the cluster wide installation we saw for etcd Operator in previous section
+. In this guide, we're installing the Service Binding Operator in `default` namespace. You can install it in the namespace you prefer to use it in
+
+Let's setup the Service Binding Operator now.
+
+. First, let's clone its repository on the host system where we're running minikube and checkout the specific tag that is currently supported by `odo link` command:
++
+[source,sh]
+----
+$ git clone https://github.com/redhat-developer/service-binding-operator/
+$ cd service-binding-operator
+$ git fetch --tags
+$ git checkout tags/v0.1.1-364 -b v0.1.1-364
+$ cd deploy
+----
+
+. Next, install its CRD (link:https://docs.openshift.com/container-platform/latest/operators/understanding/crds/crd-extending-api-with-crds.html[Custom Resource Definition]) and create link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/[ServiceAccount], https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole[Role] and link:https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding[RoleBinding] it requires:
++
+[source,sh]
+----
+$ kubectl create -f crds/apps.openshift.io_servicebindingrequests_crd.yaml
```

Comment:
could we just create it straight from github without cloning the whole source code?
```
kubectl create -f https://raw.githubusercontent.com/redhat-developer/service-binding-operator/v0.1.1-364/deploy/crds/apps.openshift.io_servicebindingrequests_crd.yaml
```

Reply:
Sure. :+1: